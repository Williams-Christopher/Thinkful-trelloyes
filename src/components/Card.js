import React from 'react';

import './Card.css';

function Card(props) {
    console.log(props);
    return (
        // <div className='Card' key={props.key}>
        <div className='Card'>
            <h3>{props.title}</h3>
            <p>{props.content}</p>
            <button type='submit' name='cardDelete' onClick={() => props.onDeleteList(props.id)}>Delete</button>
        </div>
    )
};

export default Card;