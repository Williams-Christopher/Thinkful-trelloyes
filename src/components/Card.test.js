import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Card from './Card';
import { exportAllDeclaration } from '@babel/types';

describe('Card Component Testsuite', () => {
    it('Mounts without errors', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('Renders the UI as expected', () => {
        const tree = renderer.create(<Card />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});