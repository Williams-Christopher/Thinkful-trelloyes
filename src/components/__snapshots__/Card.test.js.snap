// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Testsuite Renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3 />
  <p />
</div>
`;
